api/src/main/java/org/openmrs/module/sync2/api/model/ParentObjectHashcode.java

```diff
@@ -0,0 +1,64 @@
+package org.openmrs.module.sync2.api.model;
+
+import org.openmrs.BaseOpenmrsData;
+
+import java.util.Objects;
+
+public class ParentObjectHashcode extends BaseOpenmrsData {
+
+	private Integer id;
```

Comment:
Is this id of hashcode entity itself or of local object? In order to search through hashcodes stored on local instance we were supposed to save local objects ids. Did you solve it some other way?

Reply:
This is the database id. In order to search hashcode we use the UUID of the object.